adala/runtimes/_litellm.py

```diff
@@ -191,27 +213,43 @@ def record_to_record(
                 # extra inference params passed to this runtime
                 **self.model_extra,
             )
+            usage = completion.usage
+            dct = response.dict()
+        except IncompleteOutputException as e:
+            usage = e.total_usage
+            dct = _format_error_dict(e)
+            print_error(f"Inference error {dct['_adala_message']}: Incomplete output. Increase max_tokens.")
+            tb = traceback.format_exc()
+            logger.error(tb)
```

Comment:
to improve code redability, let's move these blocks in a separate function, and probably we don't need double logging:
```
def _log_llm_exception(self, e):
     dct = _format_error_dict(e)
     base_error = f"Inference error {dct['_adala_message']}: Incomplete output. Increase max_tokens."
     tb = traceback.format_exc()
     logger.error(f'{base_error}\nTraceback:\n{tb}')
```